packages/core/src/interpreter.ts

```diff
@@ -123,6 +123,8 @@ export class Interpreter<
 
   public src?: string;
 
+  public root: any;
```

Comment:
I don't think we'd want `root` here, right?:

- we're only assigning to `this.system.root`
- since `this` _is_ the root, referencing `interpreter.root` would be the same thing as just referencing `interpreter`

Reply:
`AnyActorRef` for now